app/config/env.php

```diff
@@ -82,3 +82,20 @@
     $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/cache_pool'));
     $loader->load($pool . '.yml');
 }
+
+// Yoochoose settings
+if ($value = getenv('YOOCHOOSE_CUSTOMER_ID')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.customer_id', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_LICENSE_KEY')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.license_key', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_SERVER_URI')) {
+    $container->setParameter('ez_recommendation.default.server_uri', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_INCLUDED_CONTENT_TYPES')) {
```

Comment:
We'll see tomorrow, but might be blocker here, as env variables will only be able to inject scalar values.
If Studio-demo had a sane default here, then that would have resolved the blocker (defining content type is more for the application to define anyway, not the environment).